app/mailers/jobseekers/alert_mailer.rb

```diff
@@ -30,6 +33,12 @@ def alert(subscription_id, vacancy_ids)
 
   attr_reader :subscription_id
 
+  def handle_invalid_email_exception(exception)
+    return subscription.destroy! if exception.message.match?(INVALID_EMAIL_REGEXP)
+
+    raise exception
```

Comment:
I'm not sure we should be destroying subscriptions - we can just make them inactive IMHO?